Src/Idioms/EqualityAssertion.cs

```diff
@@ -0,0 +1,364 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.Idioms
+{
+	/// <summary>
+	/// Represent an assertion which verifies that an equality method returns the correct value.
```

Comment:
Are we hereby implying that the base implementation (reference equality) is incorrect?